airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -1,100 +1,127 @@
-#      configured_catalog_path: "integration_tests/configured_catalog_get_fba_sns_performance_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_flat_file_archived_orders_data_by_order_date.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_flat_file_returns_data_by_return_date.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_merchant_cancelled_listings_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_merchant_listings_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_merchant_listings_data_back_compat.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_merchants_listings_fyp_report.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_merchant_listings_inactive_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_stranded_inventory_ui_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_xml_all_orders_data_by_order_date_general.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_ledger_detail_view_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_fba_inventory_planning_data.json"
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog_get_ledger_summary_view_data.json"
-# TODO: uncomment when Orders (or any other incremental) stream is filled with data
-#  incremental:
-#   - config_path: "secrets/config.json"
-#     configured_catalog_path: "integration_tests/configured_catalog.json"
-#     future_state_path: "integration_tests/future_state.json"
-#     cursor_paths:
-#       Orders: ["LastUpdateDate"]
-#  full_refresh:
-#   - config_path: "secrets/config.json"
-#     configured_catalog_path: "integration_tests/configured_catalog.json"
+    tests:
+      - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.2.30"
+  basic_read:
+    tests:
+      - config_path: "secrets/config.json"
+        timeout_seconds: 2400
+        expect_records:
+          path: "integration_tests/expected_records.jsonl"
+          extra_fields: no
+          exact_order: no
+          extra_records: yes
+        empty_streams:
+          - name: Orders
+            bypass_reason: "no records"
```

Comment:
why are there no records? Can you give some context that helps the next person understand why we can't have records here?

Reply:
Added TODO comment